web-components/src/components/pagination/tokens/lm-pagination-tokens.js

```diff
@@ -0,0 +1,54 @@
+/* eslint-disable @typescript-eslint/no-var-requires */
+/* eslint-disable no-undef */
+const colors = require("@momentum-ui/tokens/dist/colors.json");
+
+const pagination = {
+  prefix: "lm",
+  component: "pagination",
+  color: {
+    light: colors.gray[50].name,
+    dark: colors.gray[60].name,
+    current: {
+      light: colors.white[100].name,
+      dark: colors.white[100].name
+    }
+  },
+  "bg-current": {
+    light: colors.blue[50].name,
+    dark: colors.blue[60].name
+  },
+
+  "bg-hover": {
+    light: colors.gray[20].name,
+    dark: colors.gray[10].name
+  },
+  nav: {
+    color: {
+      light: colors.gray[90].name,
+      dark: colors.gray[40].name
+    },
+    focus: {
+      light: colors.blue[60].name,
+      dark: colors.blue[40].name
```

Comment:
available as global CSS var `--md-default-focus-outline-color` , no need to duplicate